.github/actions/setup/action.yml

```diff
@@ -4,35 +4,25 @@ inputs:
   node-version:
     description: Node.js version
     required: false
-    default: '14'
+    default: '16'
```

Comment:
issue(blocking): as discussed in https://github.com/rx-angular/rx-angular/pull/1391#pullrequestreview-1064696394, we should hold back a full Node.js 16.x migration until we have migrated to at least Angular 13.x **and** Nx 13.x.